bin/ChakraCore/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# xplat-todo: This is a skeleton make file and not used in build yet.
+# Please add this to build and fix issues.
+
+add_library (ChakraCore
```

Comment:
This is not a library but a dynamic load dll- we could change it later, but just calling it out. Same for rl